contracts/facade/FacadeAct.sol

```diff
@@ -79,116 +79,128 @@ contract FacadeAct is IFacadeAct {
                         );
                     }
                 }
-            } else if (!cache.bh.fullyCollateralized()) {
-                // backingManager.manageTokens([]);
-                return (
-                    address(cache.bm),
-                    abi.encodeWithSelector(cache.bm.manageTokens.selector, empty)
-                );
-            } else {
-                // collateralized
-
-                RevenueTotals memory revTotals = main.distributor().totals();
-
-                // check revenue traders
-                for (uint256 i = 0; i < erc20s.length; i++) {
-                    // rTokenTrader: if there's a trade to settle
-                    ITrade trade = cache.rTokenTrader.trades(erc20s[i]);
-                    if (address(trade) != address(0) && trade.canSettle()) {
-                        // cache.rTokenTrader.settleTrade(...)
-                        return (
-                            address(cache.rTokenTrader),
-                            abi.encodeWithSelector(
-                                cache.rTokenTrader.settleTrade.selector,
-                                erc20s[i]
-                            )
-                        );
-                    }
+            } else if (cache.bh.status() != CollateralStatus.DISABLED) {
+                // Only trade if basket is sound
```

Comment:
Nit: comment should be "Only trade if basket is non-DISABLED"

Reply:
We expect there to be times where 2 tokens default slightly offset, such that the first `refreshBasket` results in IFFY

Reply:
Totally, will fix it